### DISCUSSION TITLE
Example algorithms ideas

### DISCUSSION THREAD

**Discussion by suarezvictor** [JulianKemmerer/PipelineC#209] [Ideas]
Binary to decimal can be executed in a single cycle using few resources with this algorithm https://jk-jeon.github.io/posts/2022/02/jeaiii-algorithm/

**Comment by JulianKemmerer** [comment]
Wow - who would have thought there was so much to be said about optimizing binary to decimal, neat :nerd_face:

**Comment by suarezvictor** [comment]
Integer sqrt https://twitter.com/tom_verbeure/status/1327500422177726466
